### ISSUE TITLE
Definition for rule 'fb-www/extra-arrow-initializer' was not found  fb-www/extra-arrow-initializer

### ISSUE THREAD

**Issue by aeroyu** [facebook/draft-js#2680]
yarn run lint 
nvm 14.3.0
![image](https://user-images.githubusercontent.com/5063734/97149732-1dfd0d80-17a8-11eb-9657-a25693d1b959.png)

**Comment by mrkev** [comment]
Yup, I asked about it on the #help channel of the ESLint discord last week. Looks like the way around it is with some dummy, custom rules. It sounds like a good first issue if you (or anyone else) is looking for one— otherwise I'll just get to it probably this or next week.

<img width="708" alt="Screen Shot 2020-11-02 at 11 32 01 AM" src="https://user-images.githubusercontent.com/1285131/97899557-12a56700-1cff-11eb-94f0-78ec28370b0d.png">
